documentation/devonfw-ide-contribution-getting-started.asciidoc

```diff
@@ -0,0 +1,160 @@
+```
+doInstall "-" "«installation-folder»" "«tool-name»" "«tool-version»"
+```
+
+For standard installations omit `"«installation-folder»"` (use the empty string arg `""`) what will install to `${DEVON_IDE_HOME}/software/«tool-name»`.
+
+=== LICENSE
+Find the license for the new tool and add the tool to the table of `Third party components` of the link:LICENSE.asciidoc[LICENSE].
+If the license itself does not exist, add it to the end of the file as a new section.
+
+== Settings
+The https://github.com/devonfw/ide-settings[ide-settings] repository allows project-specific configurations of the IDE tools.
+For your own settings, fork the repository, adjust the configuration and specify the repository URL during installation.
+Important notes on configuration and especially configuration files can be found in the link:configuration.asciidoc[configuration].
+In the configuration files many link:variables.asciidoc[variables] can be defined.
+
+== Tipps and tricks
+
+The following sub-sections give you some tipps and tricks to boost your productivty when developing features for `devonfw-ide`.
+
+=== Symbolic Links to ease testing and commiting in Windows
+
+With https://schinagl.priv.at/nt/hardlinkshellext/linkshellextension.html[symbolic links] you can use one file in one directory in multiple other directories, without copying the whole file the the other directories. So you are actually just linked to the original file and therefore don't have to worry about differing versions of the file in your directory. This is very helpful for testing our devon-ide, because e.g. the commandlets folder for testing is not the same as the commandlets folder for git-commits. 
+Follow https://schinagl.priv.at/nt/hardlinkshellext/linkshellextension.html[these] installation and usage instructions first. 
+Later on you can make symbolic links of files from your "git folder" and add them into the folder in which the testing is supposed to take place.
+E.g. make a symbolic link of ${DEVON_IDE_HOME}/workspaces/ide/scripts/src/main/resources/scripts/command/docker and insert it into the folder ${DEVON_IDE_HOME}/scripts/command.
+Now changes you make in the first mentioned file  will directly be available in your testing environment.
+
+=== developer tools
+
+* https://addons.mozilla.org/en-US/firefox/addon/simple-tab-groups/[Simple Tab Groups]: Firefox plugin that helps to separate tabs e.g. tabs for projekt A, projekt B, travelplanning or hobbies 
+* https:[//git-]fork.com/[git-fork]: Tool for Windows and Mac that makes working with git easier and gives a better overview compared to just using a shell 
+* https://www.heise.de/tipps-tricks/SSH-unter-Windows-10-nutzen-4224757.html[Windows Terminal]: A tool that enables the user to use CLIs of various kinds like git-bash, Powershell, CMD, WSL-Shells or Putty
```

Comment:
These hints should IMHO go here instead,:
https://github.com/devonfw/ide/blob/master/documentation/advanced-tooling-generic.asciidoc

Leave the section about advanced tooling here and simply put a link to the existing [documentation](https://github.com/devonfw/ide/blob/master/documentation/advanced-tooling-generic.asciidoc).